bundles/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/discovery/BigAssFanDiscoveryService.java

```diff
@@ -115,21 +115,23 @@ protected void stopBackgroundDiscovery() {
         cancelListenerJob();
     }
 
-    private void startListenerJob() {
-        if (listenerJob == null) {
+    private synchronized void startListenerJob() {
+        logger.debug("Starting discovery listener job in {} seconds", BACKGROUND_DISCOVERY_DELAY);
```

Comment:
Why log this here when the discovery listener won't be started if listenerJob is null?

Reply:
fixed